pkg/apis/sealed-secrets/v1alpha1/sealedsecret_expansion.go

```diff
@@ -80,6 +80,19 @@ func NewSealedSecretV1(codecs runtimeserializer.CodecFactory, pubKey *rsa.Public
 	return s, nil
 }
 
+func StripLastAppliedAnnotations(annotations map[string]string) {
+	if annotations == nil {
+		return
+	}
+	keys := []string{
+		"kubectl.kubernetes.io/last-applied-configuration",
+		"kubecfg.ksonnet.io/last-applied-configuration",
```

Comment:
Good catch!

Reply:
do you know any other tool that does something similar?

Reply:
Not that I know of.
Perhaps make a regex on `*last-applied-configuration` ?

Reply:
I suspect I'd regret that choice when somebody would report bewilderment at a stripped annotation with a similar name but with a different semantics.

Reply:
Perhaps check all annotations and strip anything that contains original plain text?

Reply:
that wouldn't work with `"kubecfg.ksonnet.io/last-applied-configuration` since it's gzip compressed